Mathlib/Algebra/Star/StarAlgHom.lean

```diff
@@ -780,13 +780,17 @@ theorem refl_symm : (StarAlgEquiv.refl : A ≃⋆ₐ[R] A).symm = StarAlgEquiv.r
   rfl
 
 -- should be a `simp` lemma, but causes a linter timeout
-theorem to_ringEquiv_symm (f : A ≃⋆ₐ[R] B) : (f : A ≃+* B).symm = f.symm :=
+theorem toRingEquiv_symm (f : A ≃⋆ₐ[R] B) : (f : A ≃+* B).symm = f.symm :=
   rfl
 
+@[deprecated (since := "25-08-2025")] alias to_ringEquiv_symm := toRingEquiv_symm
+
 @[simp]
-theorem symm_to_ringEquiv (e : A ≃⋆ₐ[R] B) : (e.symm : B ≃+* A) = (e : A ≃+* B).symm :=
+theorem symm_toRingEquiv (e : A ≃⋆ₐ[R] B) : (e.symm : B ≃+* A) = (e : A ≃+* B).symm :=
```

Comment:
[AlgEquiv.toLinearEquiv_symm](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Algebra/Equiv.html#AlgEquiv.toLinearEquiv_symm) and [RingEquiv.coe_toAddEquiv_symm](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Ring/Equiv.html#RingEquiv.coe_toAddEquiv_symm) suggests that this name is backwards, and the other one should just be deprecated without replacement.